notification_endpoint.go

```diff
@@ -129,5 +131,8 @@ type NotificationEndpointService interface {
 	PatchNotificationEndpoint(ctx context.Context, id ID, upd NotificationEndpointUpdate) (NotificationEndpoint, error)
 
 	// DeleteNotificationEndpoint removes a notification endpoint by ID, returns secret fields, orgID for further deletion.
-	DeleteNotificationEndpoint(ctx context.Context, id ID) (flds []SecretField, orgID ID, err error)
+	DeleteNotificationEndpoint(ctx context.Context, id ID) (flds []*SecretField, orgID ID, err error)
+
+	// TestNotificationEndpoint tests the an endpoints connection to a 3rd party service i.e. Slack, PagerDuty etc.
+	TestNotificationEndpoint(ctx context.Context, ne NotificationEndpoint, userID ID) error
```

Comment:
A method prefix Test is like a reserved keyword to run test uniting in golang, thus might confuse people. If you can give an another name, it would be much preferred.

Reply:
Will do